src/main/java/chat/Chat.java

```diff
@@ -0,0 +1,55 @@
+package chat;
+
+import chat.commands.Command;
+import chat.exceptions.ChatException;
+import chat.tasks.TaskList;
+import chat.utils.Parser;
+import chat.utils.Storage;
+
+/**
+ * @author juzzztinsoong
+ */
+public class Chat {
+
+    private Storage storage;
+    private TaskList tasklist;
+    private boolean isExit;
+
+    /**
+     * Constructor method for Chat.
+     * @param filepath chat.txt filepath.
+     */
+    public Chat(String filepath) {
+        isExit = false;
+        storage = new Storage(filepath);
+        try {
+            tasklist = storage.loadFromFile();
+        } catch (ChatException e) {
+            System.err.println("File failed to load");
+            tasklist = new TaskList();
+        }
+    }
+
+    /**
+     * Interacts with chat, sends command and receives output.
```

Comment:
Perhaps you could specify sends command (to what) and receives output (from who / what)